Content/README.md

```diff
@@ -0,0 +1,49 @@
+# SAFE Template
+
+This template can be used to generate a full-stack web application using the [SAFE Stack](https://safe-stack.github.io/).
+
+## Install pre-requisites
+
+You'll need to install the following pre-requisites in order to build SAFE applications
+
+* [FAKE 5](https://fake.build/) installed as a [global tool](https://fake.build/fake-gettingstarted.html#Install-FAKE)
+* The [Yarn](https://yarnpkg.com/lang/en/docs/install/) package manager (you an also use `npm` but the usage of `yarn` is encouraged).
+* [Node LTS](https://nodejs.org/en/download/) installed for the front end components.
+* If you're running on OSX or Linux, you'll also need to install [Mono](https://www.mono-project.com/docs/getting-started/install/).
+
+## Work with the application
+
+To concurrently run the server and the client components in watch mode use the following command:
+
+```bash
+fake build -t Run
+```
+
+//#if (deploy == "docker")
```

Comment:
I would personally remove the docker and app service bits - you need to really read the docs to understand how to use them both. Perhaps replace with a couple of links to those pages, or just remove.